### PR TITLE
update readme and license file

### PR DESCRIPTION
**Pull request by ElectricMaxxx**
| Q             | A
| ------------- | ---
| Branch?       | "master" for new features / the branch of the current release for fixes
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | #9 
| License       | MIT
| Doc PR        | reference to the documentation PR, if any

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,14 +1,20 @@
-Copyright (c) 2011-2017 Eric GELOEN
```
Comment (kunicmarko20) [inline LICENSE]: Should we still keep his name?

Comment (ElectricMaxxx) [reply LICENSE]: mhh, yea .., yes.

File: LICENSE [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
+Copyright (c) 2018-2018 Eric GELOEN
```
Comment (stof) [inline LICENSE]: I think we should have it like that:

```
Copyright (c) 2018-2018 <new names>
Copyright (c) 2011-2017 Eric GELOEN
```

as done in Swiftmailer when changing maintainers

File: README.md [hunk 3]
```diff
@@ -1,34 +1,35 @@
...
+------------
 
 The bundle provides a [CKEditor](http://ckeditor.com/) integration for your Symfony Project. It automatically registers
 the new `ckeditor` form type which can be easily as well as highly configured.
 
-## Documentation
+Documentation
+-------------
+
+Please, read the [official documentation](http://symfony.com/doc/master/bundles/FOSCKEditorBundle/index.html).
```
Comment (stof) [inline README.md]: HTTPS please

File: README.md [hunk 4]
```diff
@@ -1,34 +1,35 @@
...
-[![Dependency Status](http://www.versioneye.com/php/egeloen:ckeditor-bundle/badge.svg)](http://www.versioneye.com/php/egeloen:ckeditor-bundle)
-
-[![Latest Stable Version](https://poser.pugx.org/egeloen/ckeditor-bundle/v/stable.svg)](https://packagist.org/packages/egeloen/ckeditor-bundle)
-[![Latest Unstable Version](https://poser.pugx.org/egeloen/ckeditor-bundle/v/unstable.svg)](https://packagist.org/packages/egeloen/ckeditor-bundle)
-[![Total Downloads](https://poser.pugx.org/egeloen/ckeditor-bundle/downloads.svg)](https://packagist.org/packages/egeloen/ckeditor-bundle)
-[![License](https://poser.pugx.org/egeloen/ckeditor-bundle/license.svg)](https://packagist.org/packages/egeloen/ckeditor-bundle)
+Introduction
+------------
 
 The bundle provides a [CKEditor](http://ckeditor.com/) integration for your Symfony Project. It automatically registers
```
Comment (stof) [inline README.md]: HTTPS please

File: README.md [hunk 5]
```diff
@@ -1,34 +1,35 @@
...
+-------
 
-Please, read the [official documentation](http://symfony.com/doc/master/bundles/IvoryCKEditorBundle/index.html).
+This bundle is released under the MIT license. See the included
+[LICENSE](LICENSE) file for more information.
 
 ## Testing
+----------
 
 The bundle is fully unit tested by [PHPUnit](http://www.phpunit.de/) with a code coverage close to **100%**. To
```
Comment (stof) [inline README.md]: should be HTTPS too

**Comment by kunicmarko20** [comment]
Scrutinizer running for 50 mins https://scrutinizer-ci.com/g/FriendsOfSymfony/FOSCKEditorBundle/inspections/ff6254c0-810e-4f07-9a34-d9775326ff6e

**Pull request merged**